control/control.go

```diff
@@ -705,7 +711,22 @@ func (p *pluginControl) UnsubscribeDeps(id string) []serror.SnapError {
 	return p.subscriptionGroups.Remove(id)
 }
 
+// Checks if string is URL
+func isURL(url string) bool {
```

Comment:
Consider moving this function body to single source file (for example `/grpc/common/common.go`) instead of redeclaring it multiple times